### DISCUSSION TITLE
Formatting job chips

### DISCUSSION THREAD

**Discussion by ICmike** [bullhorn/career-portal#401] [Q&A] [answered]
Hi 

Hopefully im posting this in the correct area, I'm trying to add a job chip on the portal on a Wordpress site 

I found that i can add the job chip within the plugin options and also via the FTP on the file called app.json

I've manage to add a salary job chip i'm just stuck on how to format it with a decimal and a pound sign 

so far this is the code in the json file i know i need to add GBP to the type 

 ``` 
"jobInfoChips": ["employmentType", "salary", {
      "type": "mediumDate",
      "field": "dateLastPublished"
    }],
```

if anyone could help that would be great 

Thanks
Mike

**Comment by charlesabarnes** [comment, marked as answer]
Hello,

The option for salary needs to be an object just like dateLastPublished.   So something like the following. 

`{
     "type": "GBP",
     "field": "salary"
  }`

**Reply by ICmike** [reply]
Thanks for your quick response 

just to confirm you are saying that the whole code should look like this?

```
    "jobInfoChips": ["employmentType", "salary" {
        "type": "mediumDate",
        "field": "dateLastPublished"
      },
      {
        "type": "GBP",
        "field": "salary"
      }
    ],
```

**Reply by charlesabarnes** [reply]
The object would replace the string `"salary"`, but yes. like that.

**Reply by ICmike** [reply]
Thanks for confirming 

I initially couldn't see it, but i cleared the site cache, manual synced and its displaying now 

Thanks again
